app/templates/components/user-cards.hbs

```diff
@@ -1,5 +1,5 @@
 {{#user-filtering employees=members as |employeeslist|}}
-<div class="content-container">
+<div class="content-container font-style">
```

Comment:
Instead of having `font-style` in various places, (considering this will be applied in all the places in our project) we can wrap a div with this class for the `outlet` in application.hbs. This way, this will be applied to all children of application.